Predictable Sequencing (#567)
vatpac-technology/sops: docs/controller-skills/sequencing.md

!!! note
    The times shown at waypoints when displaying an aircraft's route are not completely accurate. They are calculated based on a combination of the aircraft's ***planned TAS, Forecast GRIB winds, and estimated climb/descent profiles***. Times shown on the BRL anchored from an aircraft to a waypoint are based on the aircraft's ***current ground speed***. As a general rule of thumb, a jet aircraft will tend to add approximately 1-2 minutes to a BRL time on descent. A propeller aircraft will not add any time on descent. Both the route time and the BRL time can provide situational awareness and more info, but use whatever is appropriate for the situation.

## Predictable Sequencing 
Predictable Sequencing is an action to delay aircraft via rerouting aircraft via pre-defined waypoints off major airways to create certain time delays while also allowing a continuous descent into the airport.

!!! phraseology
    **ARL:** "QFA11, recleared ADLIV then direct RIKNI, rest of clearance unchanged, FL130"
    **QFA11:** "Cleared ADLIV RIKNI, continue descent FL130"

> 

Reviewer: ```suggestion
!!! phraseology
    **HUO:** "JST712, recleared direct OSEMI thence RAZZI to rejoin the STAR, F130"
    **JST712:** "Recleared direct OSEMI thence RAZZI to rejoin the STAR, F130, JST71"
```

## Predictable Sequencing 
Predictable Sequencing involves ATC re-routing aircraft via pre-defined waypoints positioned off major airways to provide a certain time delay. This re-routing may be used instead of vectoring and provides flight crew with predictability of lateral path to plan their descent, in alignment with the principles of ICAO Continuous Descent Operations (CDO).

!!! phraseology
    **ARL:** "QFA11, from NONID, recleared direct ADLIV thence RIKNI, rest of clearance unchanged"
    **QFA11:** "From NONID, recleared direct ADLIV thence RIKNI, QFA11"
!!! phraseology
    **HUO:** "JST712, from AKTAM, recleared direct OSEMI thence RAZZI to rejoin the STAR"

>     **JST712:** "From AKTAM, recleared direct OSEMI thence RAZZI to rejoin the STAR, JST712"

Reviewer: ```suggestion
    **JST712:** "From AKTAM, recleared direct OSEMI thence RAZZI to rejoin the STAR, JST712"
    
Controllers are advised to annunciate new waypoints phonetically when re-routing aircraft.
```

    **BLA:** "VOZ888, Cancel hold, cleared NABBA, BULLA, TAREX, BOOIN, to rejoin the STAR, maintain F300"  
    **VOZ888:** "Cancel hold, cleared NABBA, BULLA, TAREX, BOOIN, to rejoin the STAR, maintain F300, VOZ888"

It may be prudent to use another form of delaying action, other than holding, with aircraft that appear to be unable to fly a hold (use vectors).

## Predictable Sequencing 

> Through vectoring and holding, enroute controllers can effectively create and maintain an orderly sequence of arrivals. However, these techniques require pilots to fly a variable distance additional track miles until the required sequence is achieved. Pilots use the distance remaining to calculate their TOD and to maintain an appropriate descent profile, and the unpredictability of these delaying techniques—and how many extra miles will be flown—can make it more difficult for pilots to plan a smooth and orderly descent.

Reviewer: ```suggestion
Through vectoring and holding, enroute controllers can effectively create and maintain an orderly sequence of arrivals. However, these techniques require pilots to fly variable-distance tracks until the required delay is achieved. Pilots consider track miles remaining to plan and maintain a continuous descent profile, and the unpredictability of these sequencing techniques can make this calculation more difficult.
```

> As an alternative to vectoring and holding, controllers may reroute aircraft to predefined 'Predictable Sequencing' waypoints to delay an aircraft by discrete amount of time. Rerouting the aircraft allows the pilot to calculate the distance remaining more accurately, and therefore adjust their TOD and descent profile accordingly.

Reviewer: ```suggestion
As an alternative to vectoring and holding, controllers may reroute aircraft via predefined 'Predictable Sequencing' waypoints to delay an aircraft by a discrete amount of time. Rerouting the aircraft allows the pilot to calculate the distance remaining more accurately, and therefore adjust their TOD and descent profile accordingly.

Predictable Sequencing waypoints may also be referred to as Continuous Descent Operations (CDO) waypoints.
```

<figure markdown>
![Predictable Sequencing waypoints](img/cdo_n774_m636.png){ width="500" }
  <figcaption>Predictable Sequencing waypoints</figcaption>
</figure>

!!! phraseology
    *ARL wants to delay QFA11 by 8 minutes to achieve the desired sequence. They consult the Predictable Sequencing map and identify 'ADLIV' as an appropriate delaying waypoint* 
    **ARL:** "QFA11, from NONID, recleared direct ADLIV thence RIKNI, rest of clearance unchanged"  
    **QFA11:** "From NONID, recleared direct ADLIV thence RIKNI, QFA11"

<figure markdown>
![Predictable Sequencing waypoints](img/cdo_razzi.png){ width="500" }
  <figcaption>Predictable Sequencing waypoints</figcaption>
</figure>
	
!!! phraseology
    *HUO wants to delay JST712 by 6 minutes to achieve the desired sequence. They consult the Predictable Sequencing map and identify 'OSEMI' as an appropriate delaying waypoint* 
    **HUO:** "JST712, from AKTAM, recleared direct OSEMI thence RAZZI to rejoin the STAR"  
    **JST712:** "From AKTAM, recleared direct OSEMI thence RAZZI to rejoin the STAR, JST712" 
	
!!! tip

>     As pilots are unlikely to be familiar with these waypoints, controllers should consider spelling the waypoints phonetically to assist pilots identify the waypoint in their FMC.

Reviewer: ```suggestion
    As pilots are likely to be unfamiliar with these waypoints, controllers should consider spelling them phonetically to ensure correct data entry in their FMC.
```
